### PR TITLE
[Pipeline] dailyBuild: bump version and sign

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description

azure-powershell - daily-build targets to build codes in master branch every day so that we can fix issues before release.
Pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=608510&view=results

There are two main steps in this pipeline:
- Bump Version
- Sign dlls and publish artifacts

After build completion, azure-powershell - daily-smoke-test will be triggered, which runs smoke test to validate modules.
Pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=608909&view=results

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by BethanyZhou** [comment]
@isra-fel Could you please help to review this pr?

File: .azure-pipelines/daily-build.yml [hunk 1]
```diff
@@ -9,20 +9,148 @@ schedules:
...
+        ./tools/RunVersionController.ps1  -MonthName $(MonthName) -Year $(Year) `
+        Exit"
+        dotnet tool run pwsh -c $command  
+
+  - task: PowerShell@2
+    displayName: 'Push to Branch internal/dailybuild'
+    inputs:
+      targetType: 'inline'
+      script: |
+        git config user.email "bez@microsoft.com"
```
Comment (PhoenixHe-msft) [inline .azure-pipelines/daily-build.yml:67]: Why to use your own email?

Comment (PhoenixHe-msft) [reply .azure-pipelines/daily-build.yml:67]: Suggest to add 

Set-PSDebug -Trace 1

Comment (BethanyZhou) [reply .azure-pipelines/daily-build.yml:67]: Cause the github access token is from me , I haven't found a group token for this yet.

File: tools/Az/Az.psd1 [hunk 2]
```diff
@@ -57,7 +57,6 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.1.2'; },
                @{ModuleName = 'Az.Aks'; RequiredVersion = '2.0.0'; }, 
                @{ModuleName = 'Az.AnalysisServices'; RequiredVersion = '1.1.4'; }, 
                @{ModuleName = 'Az.ApiManagement'; RequiredVersion = '2.1.0'; }, 
-               @{ModuleName = 'Az.AppConfiguration'; RequiredVersion = '1.0.0'; },
```
Comment (isra-fel) [inline tools/Az/Az.psd1:60]: Can recover this after official release

File: tools/VersionController/MinimalVersion.csv [hunk 3]
```diff
@@ -1,2 +1 @@
-"Module","Minimal Version"
-"Az.AppConfiguration","1.0.0"
\ No newline at end of file
+"Module","Minimal Version"
```
Comment (isra-fel) [inline tools/VersionController/MinimalVersion.csv:1]: Same

File: tools/RunVersionController.ps1 [hunk 4]
```diff
@@ -188,6 +194,15 @@ switch ($PSCmdlet.ParameterSetName)
     "ReleaseAz"
```
Comment (isra-fel) [inline tools/RunVersionController.ps1:194]: inveestigate how to use switch

File: tools/RunVersionController.ps1 [hunk 5]
```diff
@@ -5,9 +5,15 @@
 
 [CmdletBinding(DefaultParameterSetName="ReleaseAz")]
 Param(
-    [Parameter(ParameterSetName='ReleaseAz', Mandatory = $true)]
+    [Parameter(ParameterSetName='ReleaseAz', Mandatory = $false)]
     [string]$Release,
 
+    [Parameter(ParameterSetName='ReleaseAz', Mandatory = $false)]
```
Comment (isra-fel) [inline tools/RunVersionController.ps1:11]: Use different parameter set

File: .azure-pipelines/daily-build.yml [hunk 6]
```diff
@@ -9,20 +9,148 @@ schedules:
...
+
+  - task: PowerShell@2
+    displayName: 'Clean artifacts folder'
+    inputs:
+      targetType: 'inline'
+      script: |
+        Remove-Item -Force -Recurse -Path artifacts
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Build for Test'
```
Comment (isra-fel) [inline .azure-pipelines/daily-build.yml:84]:
```suggestion
    displayName: 'Build after version bump up'
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: .azure-pipelines/daily-build.yml [hunk 7]
```diff
@@ -9,20 +9,148 @@ schedules:
...
+        dotnet tool run pwsh -c $command  
+
+  - task: PowerShell@2
+    displayName: 'Push to Branch internal/dailybuild'
+    inputs:
+      targetType: 'inline'
+      script: |
+        git config user.email "bez@microsoft.com"
+        git config user.name "azuresdkci"
+        git remote set-url origin https://$(Token)@github.com/Azure/azure-powershell.git
```
Comment (isra-fel) [inline .azure-pipelines/daily-build.yml:69]: Try to use bot's token
```suggestion
        git remote set-url origin https://$(Token)@github.com/Azure/azure-powershell.git
```

File: .azure-pipelines/daily-build.yml (hunk 1 above)
Comment (isra-fel) [reply .azure-pipelines/daily-build.yml:67]: Use bot's email

**Comment by BethanyZhou** [comment]
Successfully re-tested after polish codes: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=622396&view=results

**Review by isra-fel — APPROVED** [review]

**Pull request merged**